screenpipe-app-tauri/src-tauri/src/main.rs

```diff
@@ -204,6 +325,48 @@ async fn main() {
                         println!("quit clicked");
                         app_handle.exit(0);
                     }
+                    "start_recording" => {
+                        tokio::task::block_in_place(move || {
+                            Handle::current().block_on(async move {
```

Comment:
`tauri::async_runtime::spawn(async move {` might be better (not sure why we used tokio before)